code/game/objects/items/implants/implant_chem.dm

```diff
@@ -39,10 +39,6 @@
 		return 0
 	var/mob/living/carbon/R = imp_in
 	var/injectamount = null
-	if (cause == "action_button")
```

Comment:
You made it not inject anything ever, while also leaving the action button in place. 

I'm not sure how this passed on /tg/, maybe I'm missing something. Have you tested this PR?
:thinking: